### PR TITLE
Added Twitter Bootstrap gem

### PR DESCRIPTION
**Pull request by alan24**
I added the twitter bootstrap gem. 
Now, I am trying to figure out the style of the Sass 7-1 pattern. 

I cloned the Sass 7-1 pattern using this github: https://github.com/HugoGiraudel/sass-boilerplate.
I am not sure if this is correct. 
I am still reading up on the Sass 7-1 pattern and trying to understand it.

### CODE REVIEW

File: sass-boilerplate [hunk 1]
```diff
@@ -0,0 +1 @@
+Subproject commit b072e79e050e36178c00eb979a5bd466c9496c77
```
Comment (KaoruDev) [inline sass-boilerplate]: @alan24 can you double check this? I'm not sure what this is. Are you trying to clone down sass-boilerplate repo?

Comment (KaoruDev) [inline sass-boilerplate]: NM i saw your message! `Git` ignores empty directories in your project. So a `hack` around it is to create an empty file. `Rails` does this by stuffing empty directories with a `.keep` file.

**TLDR** :
`touch app/assets/base/.keep` and then commit that.

SO question: http://stackoverflow.com/questions/17928581/random-concerns-folders-and-keep-files

Comment (KaoruDev) [inline sass-boilerplate]: image

**Comment by alan24** [comment]
Yes, I was trying to clone down the sass-boilerplate. 
I just deleted the clone and just added the 7 empty folders inside the folder assets with the .keep file.
I'm not sure if our team will need all the 7 folders now. Also, I added the master.scss in the stylesheets folder. I just need to push my commit. 

Edit: I just pushed my code here.

**Comment by KaoruDev** [comment]
opps could you please move the new folders into `app/assets/stylesheets/*` sorry i think I led you in the wrong direction =(

**Comment by alan24** [comment]
It's ok. 
I just moved the 7 folders to the assets/stylesheets folder.
Each folder just contains a .keep. They are empty for now until we start working on the style of our chess application.
